meetings/agenda.md

```diff
@@ -6,3 +6,15 @@
 Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to Discuss
+
+### Default git branching for new majors
+
+When creating a new major, the relevant branch is created; it is also set as the default branch for the repository. However, there can be a significant gap between branch creation and the first release of that major version.
+
+Contributors and renovate, will attempt to make bugfixes to this new major when they equally apply to the previous (current latest release) major.
+
+I propose that the default branch should only be changed to the new major branch, once a release has been tagged for that new major.
```

Comment:
We might also want to talk about merge-up strategies. IMHO, until new major is not the default branch, merge-ups should be applied to both next minor and next major until default branch is being changed.